python/neuroglancer/pipeline/task_queue.py

```diff
@@ -172,7 +179,7 @@ def insert(self, task):
         }
 
         def cloud_insertion(api):
-            api.insert(
+            api.tasks().insert(
```

Comment:
This was one of the several lame choices I had to support the one use of `api.taskqueues()`